alphabet-cipher/test/alphabet_cipher/coder_test.clj

```diff
@@ -5,47 +5,31 @@
 
 (deftest test-encode
   (testing "can encode a message with a secret keyword"
-     (def v  (encode "vigilance" "meetmeontuesdayeveningatseven"))
-     (println v)
+;     (def v  (encode "vigilance" "meetmeontuesdayeveningatseven"))
```

Comment:
When you create a PR (e.g. about MagicSquare), it shouldn't have changes from other problems (e.g. alphabet cipher). The PR's files should only have changes on MagicSquare.